Jenkinsfile

```diff
@@ -689,7 +648,10 @@ pipeline {
                             filename 'Dockerfile.centos.7'
                             dir 'utils/docker'
                             label 'docker_runner'
-                            additionalBuildArgs "-t ${sanitized_JOB_NAME}-centos7 " + '$BUILDARGS'
+                            additionalBuildArgs "-t ${sanitized_JOB_NAME}-centos7 " +
+                                '$BUILDARGS_QB_TRUE' +
```

Comment:
Must have been a test that did not get backed out.  Will fix.

Reply:
Looking at it again in context.   As this is an advisory build of DAOS with CLANG, there is no reason not to use quickbuild to speed up the total build process.

Reply:
But you are not just refactoring the `Jenkinsfile` any more but rather changing the behaviour of the pipeline -- inconsistently as you have not applied this change to the other "advisory" stages.

I would suggest that this refactor PR should not change the result of the pipeline but that those kinds of pipeline changes should be in a new PR that can be reviewed by the stakeholders of those changes in the context of those change and not buried in a big refactoring PR so that the outcome of those changes are buried.

Reply:
None of the other advisory stages currently support the quickbuild option.  Only CentOS builds.

I can change it though.

Reply:
Yeah, I suppose.  That is going to change though, with the PR(s) that I am immersed in right now, to be able to build and test Leap 15 in CI, so I have my mind wrapped around the fact that QB will be applicable to Leap 15 very soon too.